src/lib/components/video/ViewVideo.svelte

```diff
@@ -148,12 +148,12 @@
               <track kind="captions" />
             </video>
           {:else}
-            <section>
+            <section class="flex justify-center">
               <iframe
                 id="player"
                 type="text/html"
-                width="456"
-                height="342"
+                width={window.innerWidth >= 540 ? '456' : window.innerWidth >= 425 ? '342' : '247'}
```

Comment:
@jacobbowdoin Please tell me if this is good or what should I do instead? I'm making this because iframes are not responsive as the videos are, and I need iframes in order to show the videos when a user uses a video link instead of storing their videos in our firebase storage.